pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go

```diff
@@ -102,14 +102,23 @@ type APIServiceDefinitions struct {
 	Required []APIServiceDescription `json:"required,omitempty"`
 }
 
-// ClusterServiceVersionSpec declarations tell the OLM how to install an operator
-// that can manage apps for given version and AppType.
+// NativeAPI describes the GVK of a Kubernetes resource that is required by an
+// Operator, but must be provided by the underlying cluster and not an extension.
+type NativeAPI struct {
```

Comment:
There might be an existing struct that can be used here that is JSON-serializable.

Reply:
Like https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1#GroupVersionKind ?